app/filtering.js

```diff
@@ -196,7 +196,7 @@ function registerForBeforeSendHeaders (session) {
           requestHeaders['Cookie'] = undefined
         }
         if (requestHeaders['Referer'] &&
-            !refererExceptions.includes(parsedUrl.hostname)) {
+            !refererExceptions.includes(parsedUrl.hostname) && !refererExceptions.includes(getBaseDomain(parsedUrl.hostname))) {
```

Comment:
cc @diracdeltas This task is good on its own without this. We don't need both checks here.  Would you prefer we update `refererExceptions` to the base domains and use `getBaseDomain` here or would you prefer we use the more narrow exceptions with not modifying `refererExceptions` and not using `getBaseDomain`?